code/modules/projectiles/projectile/bullets/rifle.dm

```diff
@@ -58,20 +58,126 @@
 	speed = 0.4
 	dismemberment = 2 //It's a budget sniper rifle.
 	armour_penetration = 20 //A bit better versus armor. Gets past anti laser armor or a vest, but doesnt wound proc on sec armor.
 	wound_bonus = 10
+	bare_wound_bonus = 20
+	embedding = list("embed_chance" = 80, "fall_chance" = 1, "jostle_chance" = 3, "ignore_throwspeed_threshold" = TRUE, "pain_stam_pct" = 0.4, "pain_mult" = 3, "jostle_pain_mult" = 2,"rip_time" = 14)
+	embed_falloff_tile = -3
+	shrapnel_type = /obj/item/ammo_casing/rebar/syndie
+
+/obj/projectile/bullet/rebar/zaukerite
+	name = "zaukerite shard"
+	icon_state = "rebar"
+	damage = 60
+	speed = 0.4
+	dismemberment = 2
+	damage_type = TOX
+	eyeblur = 5
+	armour_penetration = 15 // not nearly as good, as its not as sharp.
+	wound_bonus = 0
+	bare_wound_bonus = 20
+	embedding = list("embed_chance" = 90, "fall_chance" = 1, "jostle_chance" = 5, "ignore_throwspeed_threshold" = TRUE, "pain_stam_pct" = 0.8, "pain_mult" = 6, "jostle_pain_mult" = 2, "rip_time" = 30)
+	embed_falloff_tile = -2 // very spiky.
+	shrapnel_type = /obj/item/ammo_casing/rebar/zaukerite
+
+/obj/projectile/bullet/rebar/hydrogen
+	name = "metallic hydrogen bolt"
+	icon_state = "rebar"
+	damage = 40
+	speed = 0.6
+	dismemberment = 0 //goes through clean.
+	damage_type = BRUTE
+	armour_penetration = 30 //very pointy.
+	projectile_piercing = PASSMOB //felt this might have been a nice compromise for the lower damage for the difficulty of getting it
+	wound_bonus = -15
 	bare_wound_bonus = 10
-	embedding = list(embed_chance=80, fall_chance=1, jostle_chance=3, ignore_throwspeed_threshold=TRUE, pain_stam_pct=0.4, pain_mult=3, jostle_pain_mult=2, rip_time=14)
+	embedding = list("embed_chance" = 50, "fall_chance" = 2, "jostle_chance" = 3, "ignore_throwspeed_threshold" = TRUE, "pain_stam_pct" = 0.6, "pain_mult" = 4, "jostle_pain_mult" = 2, "rip_time" =18)
+	embed_falloff_tile = -3
+	shrapnel_type = /obj/item/ammo_casing/rebar/hydrogen
+
+/obj/projectile/bullet/rebar/healium
+	name = "healium bolt"
+	icon_state = "rebar"
+	damage = 0
+	speed = 0.4
+	dismemberment = 0
+	damage_type = BRUTE
+	armour_penetration = 100
+	wound_bonus = -100
+	bare_wound_bonus = -100
+	embedding = list(embed_chance = 0)
 	embed_falloff_tile = -3
-	shrapnel_type = /obj/item/stack/rods
+	shrapnel_type = /obj/item/ammo_casing/rebar/healium
+
+/obj/projectile/bullet/rebar/healium/on_hit(atom/target, blocked = 0, pierce_hit)
+	. = ..()
+	if(!isliving(target))
+		return BULLET_ACT_HIT
+	else
+		var/mob/living/breather = target
+		breather.SetSleeping(3 SECONDS)
+		breather.adjustFireLoss(-30, updating_health = TRUE, required_bodytype = BODYTYPE_ORGANIC)
+		breather.adjustToxLoss(-30, updating_health = TRUE, required_biotype = BODYTYPE_ORGANIC)
+		breather.adjustBruteLoss(-30, updating_health = TRUE, required_bodytype = BODYTYPE_ORGANIC)
+		breather.adjustOxyLoss(-30, updating_health = TRUE, required_biotype = BODYTYPE_ORGANIC, required_respiration_type = ALL)
+
+	return BULLET_ACT_HIT
```

Comment:
It can be even BETTER
```suggestion
	if(!isliving(target))
		return BULLET_ACT_HIT
	var/mob/living/breather = target
	breather.SetSleeping(3 SECONDS)
	breather.adjustFireLoss(-30, updating_health = TRUE, required_bodytype = BODYTYPE_ORGANIC)
	breather.adjustToxLoss(-30, updating_health = TRUE, required_biotype = BODYTYPE_ORGANIC)
	breather.adjustBruteLoss(-30, updating_health = TRUE, required_bodytype = BODYTYPE_ORGANIC)
	breather.adjustOxyLoss(-30, updating_health = TRUE, required_biotype = BODYTYPE_ORGANIC, required_respiration_type = ALL)

	return BULLET_ACT_HIT
```

Reply:
<img width="424" alt="image" src="https://github.com/tgstation/tgstation/assets/69398298/4487046f-c48b-4656-8e11-f1d0ca6a88bf">
Github really ruins this so ill take your word for it!